tests/test_cli.py

```
@@ -3061,23 +3009,89 @@ def test_print_environment_summary_windows(self, mock_print):
 
-        # Verify results
-        mock_print.assert_any_call("\n🖥️  Environment Summary:")
-        mock_print.assert_any_call("   Platform: darwin")
-        mock_print.assert_any_call("   Python: 3.12.3")
         mock_print.assert_any_call("   E2EE Support: ✅ Available and installed")
 
 
+class TestProviderAwareE2EEGuidance(unittest.TestCase):
+    """Test CLI guidance changes by provider."""
+
+    def _make_caps(self, **overrides):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_make_caps`

(ANN202)